src/main/java/com/concurrency/Counter.java

```diff
@@ -0,0 +1,14 @@
+package com.concurrency;
+
+public class Counter {
+    public static final int END_POINT = 100;
```

Comment:
let's make this field private and create a method that will check if the `value` is less than `END_POINT`